README.md

```diff
@@ -45,16 +45,24 @@ For a streamlined setup using Docker:
 
 2. **Connect to Open WebUI:**
 
-   - Navigate to the **Settings > Connections > OpenAI API** section in Open WebUI.
-   - Set the API URL to `http://localhost:9099` and the API key to `0p3n-w3bu!`. Your pipelines should now be active.
+   - Navigate to the **Admin Panel > Settings > Connections > OpenAI API** section in Open WebUI.
+   - Set the API URL to `http://localhost:9099` and the API key to `0p3n-w3bu!` (you can modify the API key in the `config.py` file). Your pipelines should now be active.
```

Comment:
Modifying the env var `PIPELINES_API_KEY` should be the recommended way to edit the default API key.